Dockerfile.test-infra

```diff
@@ -1,3 +1,4 @@
+FROM quay.io/ocpmetal/bm-inventory:latest AS inventory
```

Comment:
This should not always be "latest"
The tag should be passed as a m argument

Reply:
this should be param

Reply:
look here https://github.com/openshift/assisted-test-infra/pull/12/files

Reply:
Fixed.

Reply:
@ronniel1 @tsorya is this something that should be addressed now or can wait?